docs/netlify/_redirects

```diff
@@ -0,0 +1,530 @@
+/docs /docs/javascript-data-grid
+/docs/latest /docs/javascript-data-grid
+/docs/14.6 /docs/14.6/javascript-data-grid
+/docs/14.5 /docs/14.5/javascript-data-grid
+/docs/14.4 /docs/14.4/javascript-data-grid
+/docs/14.3 /docs/14.3/javascript-data-grid
+/docs/14.2 /docs/14.2/javascript-data-grid
+/docs/14.1 /docs/14.1/javascript-data-grid
+/docs/14.0 /docs/14.0/javascript-data-grid
+/docs/13.0 /docs/13.0/javascript-data-grid
+/docs/12.4 /docs/12.4/javascript-data-grid
+/docs/12.3 /docs/12.3/javascript-data-grid
+/docs/12.2 /docs/12.2/javascript-data-grid
+/docs/12.1 /docs/:major.:minor/javascript-data-grid
```

Comment:
Those redirects should be dynamic - non-version-aware as possible. I guess one-liner as `/docs/:major.:minor /docs/:major.:minor/javascript-data-grid`?
```suggestion
/docs/14.6 /docs/14.6/javascript-data-grid
/docs/14.5 /docs/14.5/javascript-data-grid
/docs/14.4 /docs/14.4/javascript-data-grid
/docs/14.3 /docs/14.3/javascript-data-grid
/docs/14.2 /docs/14.2/javascript-data-grid
/docs/14.1 /docs/14.1/javascript-data-grid
/docs/14.0 /docs/14.0/javascript-data-grid
/docs/13.0 /docs/13.0/javascript-data-grid
/docs/12.4 /docs/12.4/javascript-data-grid
/docs/12.3 /docs/12.3/javascript-data-grid
/docs/12.2 /docs/12.2/javascript-data-grid
/docs/12.1 /docs/:major.:minor/javascript-data-grid
```